api/schema.py

```diff
@@ -33,3 +38,41 @@ def _get_response_bodies(self, direction: str = "response") -> dict[str, Any]:
                 status_code="400",
             )
         return bodies
+
+
+class TeacherNonTeachingHelpProvidedFieldFix(OpenApiSerializerFieldExtension):
+    """
+    This class automatically extends schema for NonTeachingHelpSerializerField,
+    showing this object properly in docs.
```

Comment:
```suggestion
    Extends schema for `NonTeachingHelpSerializerField` for proper representation in docs.
```
So it fits on one line and we don't have to follow the convention with summary line and the rest on other lines :)
Same for  `TeacherPeerSupportFieldFix` class below.

Reply:
api/schema.py:46:100: E501 Line too long (101 > 99 characters)

Reply:
I think this convention should not to be followed if it doesn't fit into one-line, but it should be a one-line by design

Reply:
But I still reworded it here and below

Reply:
> I think this convention should not to be followed if it doesn't fit into one-line, but it should be a one-line by design

OK :)